core/templates/css/oppia-material.css

```diff
@@ -695,6 +695,7 @@ textarea.cdk-textarea-autosize-measuring-firefox {
   font-family: Roboto, "Helvetica Neue", sans-serif; }
 
 .mat-checkbox-layout .mat-checkbox-label {
+  font-weight: normal;
```

Comment:
oppia-material.css should not be modified... You will have to either use this same selector in oppia.css or individually do this in each component. This is done to preserve the default values.

Reply:
okay sure